### PR TITLE
add examples of while loops

### LINKED ISSUE #10: Add example: while loop

### PR DESCRIPTION
**Pull request by km1chno**
Resolves #10 

TODO:
- file extensions

### CODE REVIEW

File: examples/incorrect/while.ag
Comment (mhorod) [inline examples/incorrect/while.ag:1]: Please split these examples across separate files

**Review by mhorod — COMMENTED** [review]
> file extensions

We can use `.ccp`

**Review by mhorod — CHANGES_REQUESTED** [review]
.

File: examples/incorrect/while.ag
Comment (km1chno) [reply examples/incorrect/while.ag:1]: done

**Review by mhorod — APPROVED** [review]

**Pull request merged**